src/core/Exception.pm6

```diff
@@ -1528,6 +1528,14 @@ my class X::Syntax::Comment::Embedded does X::Syntax {
     method message() { "Opening bracket required for #` comment" }
 }
 
+my class X::Syntax::Pod::Declarator::Leading does X::Syntax does X::Pod {
+    method message() { "Opening bracket required for #| declarator block" }
+}
+
+my class X::Syntax::Pod::Declarator::Trailing does X::Syntax does X::Pod {
```

Comment:
I wonder if perhaps these two exception classes would work better as a single one with an attribute to declare what kind of comment it is; probably doesn't want to be merged with the one above since that's not Pod-related

Reply:
I'll do that.

Reply:
Hm, I have to learn more about Exception classes to do that.

Reply:
should be enough to add a named argument to the `<.typed_panic ...>` and have an attribute in the exception class, then use that attribute in the message method to change how it is displayed

Reply:
Okay.